app/assets/css/main.css

```
@@ -83,68 +83,68 @@ body {
-.prismic-rich-text h3,
-.prismic-rich-text h4,
-.prismic-rich-text h5,
-.prismic-rich-text h6 {
+.rich-text h1,
+.rich-text h2,
+.rich-text h3,
+.rich-text h4,
+.rich-text h5,
+.rich-text h6 {
   @apply font-black uppercase mt-8 mb-4;
   text-shadow: 2px 2px 0px rgba(0, 0, 0, 1);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)